doc/nrf/doc_build.rst

```diff
@@ -62,30 +61,17 @@ Complete the following steps to build the documentation output:
 
            cd ~/ncs/nrf/doc
 
-     #. Create a folder named ``_build`` and enter it:
-
-        .. code-block:: console
-
-           mkdir -p _build && cd _build
-
-#. Load the environment setting for Zephyr builds.
-
-   * On Windows:
-
-        .. code-block:: console
-
-           ..\..\..\zephyr\zephyr-env.cmd
-   * On Linux or macOS:
+#. Generate the Ninja build files:
 
         .. code-block:: console
 
-           source ../../../zephyr/zephyr-env.sh
+           cmake -GNinja -B_build .
```

Comment:
Why are we changing this command? The Zephyr instructions (at least in our fork) for building the docs haven't changed.

And so zephyr-env.sh isn't needed anymore? Or not supported anymore?
That probably means https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#setting-up-the-command-line-build-environment must be changed as well.

Reply:
> 
> 
> Why are we changing this command? The Zephyr instructions (at least in our fork) for building the docs haven't changed.
> 


> And so zephyr-env.sh isn't needed anymore? Or not supported anymore?

It is not needed anymore.
Going to send a PR upstream as well, cause it's also no longer needed upstream.
But it is still supported.

> That probably means https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#setting-up-the-command-line-build-environment must be changed as well.

Yes, and that will be updated as part of this:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2232
where samples will be updated.